controller/stats_reporter.go

```diff
@@ -99,49 +99,49 @@ func init() {
 		ItemsInList: stats.Float64(
 			"reflector_items_in_list",
 			"How many items an API list returns to the reflectors",
-			stats.UnitNone,
+			stats.UnitDimensionless,
 		),
 		// TODO(mattmoor): This is not in the latest version, so it will
 		// be removed in a future version.
 		ItemsInMatch: stats.Float64(
 			"reflector_items_in_match",
 			"",
-			stats.UnitNone,
+			stats.UnitDimensionless,
 		),
 		ItemsInWatch: stats.Float64(
 			"reflector_items_in_watch",
 			"How many items an API watch returns to the reflectors",
-			stats.UnitNone,
+			stats.UnitDimensionless,
 		),
 		LastResourceVersion: stats.Float64(
 			"reflector_last_resource_version",
 			"Last resource version seen for the reflectors",
-			stats.UnitNone,
+			stats.UnitDimensionless,
 		),
 		ListDuration: stats.Float64(
 			"reflector_list_duration_seconds",
 			"How long an API list takes to return and decode for the reflectors",
-			stats.UnitNone,
+			stats.UnitDimensionless,
 		),
 		Lists: stats.Int64(
 			"reflector_lists_total",
 			"Total number of API lists done by the reflectors",
-			stats.UnitNone,
+			stats.UnitDimensionless,
 		),
 		ShortWatches: stats.Int64(
 			"reflector_short_watches_total",
 			"Total number of short API watches done by the reflectors",
-			stats.UnitNone,
+			stats.UnitDimensionless,
 		),
 		WatchDuration: stats.Float64(
 			"reflector_watch_duration_seconds",
 			"How long an API watch takes to return and decode for the reflectors",
-			stats.UnitNone,
+			stats.UnitDimensionless,
```

Comment:
Probably need to fix this one too s/UnitDimensionless to "s"

Reply:
Or `UnitSeconds`

Reply:
Yeah I just updated all of "s" instances ;)